addons/hr_attendance/models/hr_employee_public.py

```diff
@@ -0,0 +1,13 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import fields, models
+
+class HrEmployeePublic(models.Model):
+    _inherit = 'hr.employee.public'
+
+    # These are required for manual attendance
+    attendance_state = fields.Selection(related='employee_id.attendance_state', readonly=True)
+    hours_today = fields.Float(related='employee_id.hours_today', readonly=True)
+    last_attendance_id = fields.Many2one(related='employee_id.last_attendance_id', readonly=True)
+    total_overtime = fields.Float(related='employee_id.total_overtime', readonly=True)
```

Comment:
I think those should at least be restricted to users of `hr_attendance.group_hr_attendance_kiosk,hr_attendance.group_hr_attendance`, otherwise anyone can check the overtime / last attendance of others which is a bit meh :smile:

Reply:
They used to be public before the overtime count task, but they are actually used when loading the attendance kiosk, currently in master the attendance app does not open without 'manager' rights, we could probably keep all fields as they were and fix those issues